sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityClient.java

```diff
@@ -200,20 +199,35 @@ public AccessToken issueToken(CommunicationUserIdentifier communicationUser,
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Response<AccessToken> issueTokenWithResponse(CommunicationUserIdentifier communicationUser,
-        List<CommunicationTokenScope> scopes, Context context) {
+        Iterable<CommunicationTokenScope> scopes, Context context) {
         Objects.requireNonNull(communicationUser);
         Objects.requireNonNull(scopes);
         context = context == null ? Context.NONE : context;
+        final List<CommunicationTokenScope> scopesInput = new ArrayList<>();
+        scopes.forEach(scope -> scopesInput.add(scope));
```

Comment:
These 2 lines can be replaced by the following code. This prevents external mutation of a list.

```java
StreamSupport.stream(scopes.spliterator(), false).collect(Collectors.toList());
```

Reply:
Done, thank you!